.github/workflows/change_task_count.yaml

```diff
@@ -0,0 +1,35 @@
+name: Change Task Count
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: What environment to change the task count for
+        options:
+          - dev
+          - staging
+      new_count:
+        description: 1 to turn on 0 to turn off
+        required: true
+      application_name:
+        type: choice
+        description: What application to adjust the task count for
+        options:
+          - ingestion
+          - rail-performance-manager
+
+jobs:
+  set_count:
+    if: |
+      ( github.event.inputs.new_count == 0 || github.event.inputs.new_count == 1)
```

Comment:
choice only seems to work with ints (at least my linter yelled at me) so i left these as a manual entry. keeping this if gate to prevent us from setting the task count to 11 or something.